pandas/_libs/ops_dispatch.pyi

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+REVERSED_NAMES: dict[str, str]
+UFUNC_ALIASES: dict[str, str]
+DISPATCHED_UFUNCS: set[str]
+
+def maybe_dispatch_ufunc_to_dunder_op(
+    self, ufunc: np.ufunc, method: str, *inputs, **kwargs
```

Comment:
maybe type self, since function is not a method

Reply:
will take some tracking down, since it is called via arraylike.array_ufunc, which doesnt type self